### DISCUSSION TITLE
IID removed, but no InstanceId or InstallationId as be implemented

### DISCUSSION THREAD

**Discussion by goriverna** [invertase/react-native-firebase#5482] [Q&A]
Hi all,

After the new updates, you have correctly removed the iid package, and correctly we can use the messaging module to retrieve FCM Token with getToken() and deleteToken() to delte it, but the iid module give us also the opportunity to get the firebase installationId (as named also as InstanceId), and now there's no module to get it. 
The installationId is very important because is the id that is used to send in-app-messaging on server side. 
You can find more info here: https://firebase.google.com/docs/projects/manage-installations#fid-iid

Pls add this information on the changelog. Thank you all for your great work

**Comment by mikehardy** [comment]
Hi there! The changelog file has an edit button at top right, you may propose any documentation PR any time using a simple web flow - I'll happily merge it if you give it a try?
